ssl-config/build.gradle

```diff
@@ -21,3 +29,45 @@ task generateCerts(type:Exec) {
 
 tasks.check.dependsOn(javadoc)
 tasks.test.dependsOn(generateCerts)
+
+dockerTestRunner {
+    // missing BouncyCastle but contains Sun libraries
+    dockerRunner name: 'openjdk-7', dockerFile: file('src/test/resources/openjdk-7.dockerfile')
+
+    // missing BouncyCastle and Sun libraries
+    dockerRunner name: 'ibmjdk-8', dockerFile: file('src/test/resources/ibmjdk-8.dockerfile')
+
+    // run only the PKCS1 tests and exclude BouncyCastle runtime dependency
+    testConfig = {
+        include('**/pkcs1/*Tests*')
```

Comment:
hmm, that's kind of dangerous: if someone (in a java IDE) renames the package, then the tests will be silently ignored.